Makefile

```diff
@@ -2,12 +2,12 @@ SHELL=/usr/bin/env bash -o errexit
 
 .PHONY: help check check-boskos check-core check-services dry-core core dry-services services all update template-allowlist release-controllers checkconfig jobs ci-operator-config registry-metadata boskos-config prow-config validate-step-registry new-repo branch-cut prow-config multi-arch-gen
 
-export CONTAINER_ENGINE ?= docker
+export CONTAINER_ENGINE ?= podman
```

Comment:
we should move this to a separate pr so that it could get proper review on this change from its owners